internal/subtitles/client.go

```
@@ -0,0 +1,182 @@
+		Link     string `json:"link"`
+		FileName string `json:"file_name"`
+	}
+	if err := json.NewDecoder(resp.Body).Decode(&parsed); err != nil {
+		return nil, fmt.Errorf("subtitles: decode download response: %w", err)
+	}
+
+	fileResp, err := c.httpClient().Get(parsed.Link)
+	if err != nil {
+		return nil, err
+	}
+	defer fileResp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `fileResp.Body.Close` is not checked

(errcheck)